src/components/donate/HeroSection.tsx

```diff
@@ -86,7 +87,7 @@ const HeroSection = () => {
                 {/* should donate links open up new tab? and are these the correct links */}
```

Comment:
Yeah that's a good idea for all external links!  I believe there may be a custom Link component that handles this automatically (choosing between opening links on the same page or in a new tab / window).